README.md

```diff
@@ -24,78 +24,71 @@ For 500 WGS [samples](misc/control_sample_IDs.txt) of the Medical Genome Referen
 
 Please refer to the [manuscript](https://doi.org/10.1186/s13073-021-00841-x) for further details.
 
+# ClinSV version 1.0
+This repository contains the source code and Docker files required to run ClinSV version 1.0. Version 1.0 only supports GRCh38 as the reference genome used. Please refer to release v0.9 to use with ClinSV with reference genome GRCh37 decoy (hs37d5)
+
 ## Download
 
-Download human genome reference data GRCh37 decoy (hs37d5):
+Download human genome reference data GRCh38:
 
 ```
-wget https://nci.space/clinsv/refdata-b37_v0.9.tar
-# check md5sum: 921ecb9b9649563a16e3a47f25954951
-tar xf refdata-b37_v0.9.tar
-refdata_path=$PWD/clinsv/refdata-b37
+wget https://nci.space/_projects/clinsv_b38/
```

Comment:
(a) this line is missing suffix `refdata-b38_v1.0.tar`
(b) we will still need the section on how to download data for hs37d5 (that is until v1.1 when we'll have the --reference param implemented)

Reply:
Thanks for picking that up, I just added the appropriate link